src/js/main.js

```diff
@@ -53,7 +53,12 @@ $(function () {
     Function to switch campuses
  */
 window.switchCampus = () => {
-    if (window.location.hash.toLowerCase() === '#chennai') {
+    if (window.location.hash.toLowerCase() === '#ap') {
+        $('#campus').text('AP Campus');
+        $('#last-update').text(lastUpdate.ap);
```

Comment:
You need to add this field in the `package.json` file
https://github.com/vatz88/FFCSonTheGo/blob/8a29469e8bd21090cc3fc23fee2a1615c10726ce/package.json#L4-L7

Reply:
And add a badge for this in the `README.md` as well.